<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>1209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>1151</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1151</width>
       <height>628</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Business Page:</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>491</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(181, 206, 227);</string>
       </property>
       <property name="title">
        <string>Select Location</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>State</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>47</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>City</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>40</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Zipcode</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>491</width>
          <height>261</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Select Location</string>
        </property>
        <widget class="QComboBox" name="stateList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>State</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>47</width>
           <height>14</height>
          </rect>
         </property>
         <property name="text">
          <string>City</string>
         </property>
        </widget>
        <widget class="QListWidget" name="cityList">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>211</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QListWidget" name="zipcodeList">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>60</y>
           <width>151</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Zipcode</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>371</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Zipcode Statistics</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string># of Businessess:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Population:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Average Income:</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="numBusinesses">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="totalPopulation">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>90</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="averageIncome">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>0</y>
        <width>301</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Top Categories</string>
      </property>
      <widget class="QTableWidget" name="topCategoriesList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>251</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>290</y>
        <width>1151</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Filter on Categories</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Businesses</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>60</y>
         <width>811</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QListWidget" name="categoryList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>211</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>36</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Select Category</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>710</y>
      <width>1151</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Popular Businesses (In zipcode)</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="popularBusinessTable">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>501</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Successful Businesses (In zipcode)</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="successfulBusinessTable">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>40</y>
       <width>541</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>CommandLinkButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
